web/themes/joinup/templates/field/field--fieldset-wrapper.html.twig

```diff
@@ -0,0 +1,65 @@
+{#
```

Comment:
This template is unnecessary having the preprocess function which can manage attributes.

Reply:
Yes, I had this comment [previously](https://github.com/ec-europa/joinup-dev/pull/873#discussion_r139714564) as well indeed. Though, I have not created a ticket for this, as here I have asked for a specific change, not a refactoring in the whole system.